doc/nrf/known_issues.rst

```diff
@@ -831,6 +831,14 @@ SoftDevice Controller
 
 .. rst-class:: v1-5-0
 
+DRGN-15465: Corrupted advertising data when :option:`CONFIG_BT_EXT_ADV` is set
+  Setting scan response data using legacy HCI commands on a build with extended advertising support corrupts parts of the advertising data.
+  Zephyr sets the complete local name in the scan response data by default, and the corrupted advertising data may confuse some scanners causing the device to not be visible.
```

Comment:
```suggestion
DRGN-15465: Corrupted advertising data when extended advertising support is enabled
  Setting scan response data using legacy HCI commands when :option:`CONFIG_BT_EXT_ADV` is set corrupts parts of the advertising data.
  Zephyr sets the complete local name in the scan response data by default.
  The corrupted advertising data might confuse some scanners, which causes the device to not be visible.
```

Reply:
Not sure I understand where the complete local name comes in here? Can you clarify this?

Reply:
I think the second sentence could be removed, it explains some of the symptoms, but the sentence above describes the whole issue.

Or it could be rephrased to say that when using BT_LE_ADV_OPT_USE_NAME the name is put in the scan response which would then be corrupted.

Complete local name is an AD (Advertising Data) type.
However the type could also be Shortened local name. So best to just say "device name" instead, which is the term used in the API.

Reply:
Yes, I'll reword it. I think it's useful to mention that this is linked to advertising the name of the device, so the users know that it is this issue that is affecting them.

Reply:
Yes, it makes sense to have that information, but there should be a clear connection - right now it doesn't say how this causes a problem.

Reply:
I just updated the message, lemme know if it makes sense to you